### PR TITLE
Document how images work on the web

### PR DESCRIPTION
**Pull request by yjbanov**
Document the nuances of displaying images on the web. A question that's particularly common is how to render cross-origin images in CanvasKit.

### CODE REVIEW

File: src/docs/development/platform-integration/web-images.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+images publicly, but is not configured with the correct CORS headers.
+
+Examples:
+
+- Using [CloudFlare Workers][18].
+- Using [Firebase Functions][19].
+
+### Use <img> in a platform view.
+
+Flutter supports embedding HTML inside the app using [HtmlElementView][17]. Use
```
Comment (johnpryan) [inline src/docs/development/platform-integration/web-images.md:128]: Filed an issue to add sample code for this: https://github.com/flutter/website/issues/5063

**Review by johnpryan — APPROVED** [review]

**Review by hterkelsen — APPROVED** [review]

File: src/docs/development/platform-integration/web-images.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+---
+
+The web supports the standard [Image][1] widget to display images. However,
+because web browsers are built to run untrusted code safely, there are certain
+limitations in what you can do with images compared to mobile and desktop
+platforms. This page explains these limitations and offers ways to work around
+them.
+
+# Background
+
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/web-images.md:14]: You must have at least one line of text between levels of headers.

File: src/docs/development/platform-integration/web-images.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+## Cross-origin images
+
+The HTML renderer can load cross-origin images without extra configuration.
+
+CanvasKit requires that the app get the bytes of the encoded image. There are
+several ways to do this, discussed below.
+
+### Host your images in a CORS-enabled CDN.
+
+Typically content delivery networks (CDN) can be configured to customize what
```
Comment (sfshaza2) [inline src/docs/development/platform-integration/web-images.md:106]: Comma after Typically.

**Review by sfshaza2 — APPROVED** [review]
Very minor nits, otherwise LGTM!

File: src/docs/development/platform-integration/web-images.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
+---
+title: Displaying images on the web.
```
[the remark this answers is not in the document]
Comment (yjbanov) [reply src/docs/development/platform-integration/web-images.md:2]: Done.

File: src/docs/development/platform-integration/web-images.md (hunk 2 above)
Comment (yjbanov) [reply src/docs/development/platform-integration/web-images.md:14]: Done.

File: src/docs/development/platform-integration/web-images.md (hunk 3 above)
Comment (yjbanov) [reply src/docs/development/platform-integration/web-images.md:106]: Done.

**Pull request merged**